### DISCUSSION TITLE
Configuration of Home Assistant InfuxDB Addon

### DISCUSSION THREAD

**Discussion by Rayby0073** [alexjustesen/speedtest-tracker#567] [Q&A] [answered]
Hello,
i want to store the data in the Home Assistant InfluxDB Addon.
But the credentials are different. 
There ist no "Org" and no "token" available.
Is it possible?

**Comment by alexdelprete** [comment, marked as answer]
From the docs:

![image](https://user-images.githubusercontent.com/7027842/235135112-499149df-12ae-4a0c-b09f-884ed9ed0482.png)
